spec/forms/assessor_interface/assessment_declaration_decline_form_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+RSpec.describe AssessorInterface::AssessmentDeclarationDeclineForm,
+               type: :model do
+  let(:session) { {} }
+  subject(:form) do
+    described_class.new(
+      session:,
+      assessment:,
+      declaration: true,
+      recommendation_assessor_note: "Note",
+    )
+  end
+
+  context "when recommendation is declined with no failure reasons" do
+    let(:assessment) { create(:assessment, :decline) }
+
+    describe "#save" do
+      context "when valid" do
+        subject(:save) { form.save }
+        it { is_expected.to be true }
+      end
+    end
+
+    it { is_expected.to validate_presence_of(:recommendation_assessor_note) }
+  end
+
+  context "when recommendation is started" do
+    let(:assessment) { create(:assessment, :started) }
+    it { is_expected.to validate_presence_of(:declaration) }
+  end
+
+  context "when recommendation is declined with failure reasons" do
+    before(:each) do
```

Comment:
```suggestion
    before do
```

It's not a problem, but as it's the default value typically it's not specified.